tg_bot/modules/locks.py

```diff
@@ -199,6 +255,16 @@ def del_lockables(bot: Bot, update: Update):
                         chat.kick_member(new_mem.id)
                         message.reply_text("Only admins are allowed to add bots to this chat! Get outta here.")
             else:
+                #allow whitelisted URLs
+                if lockable == 'url':
```

Comment:
should be in `else if` block, not in `else`

Reply:
Here I see bot locks as a very different beast, as it results in kicking a member. All other locks result in deleting a message. So there's the check for the bot lockable, and "all the other ones" (everything inside the `else`).

Inside that `else`, URL locks have this special case. If there was a photo whiltelist, I would put it as an `elif` at the same level as my URL `if`, and so on. Then at the end, the code to delete the message only needs to appear once, executed if none of the checks executed a `continue`